src/ModelManager.ts

```diff
@@ -66,6 +66,7 @@ export class ModelManager {
     private _fetchPromises: { [key: string]: Promise<Model> } = {};
     private _initPromise: any;
     private _editorClient: EditorClient | undefined;
+    public static _domain = "";
```

Comment:
@pfauchere Do we have a better way of getting domain from ModelManager into Util? It didn't feel right to move the Util functions under ModelManager due to apparently not much correlation.